### ISSUE TITLE
Release Mock v5.6

### ISSUE THREAD

**Issue by praiskup** [rpm-software-management/mock#1368]
There are several commits in `main` branch already.

**Comment by FrostyX** [comment]
https://rpm-software-management.github.io/mock/Release-Notes-5.6

**Comment by praiskup** [comment]
Would you mind pushing this to Fedora Copr and Red Hat Copr?

**Comment by FrostyX** [comment]
Pushed to infratags and deployed to production on Monday

**Comment by FrostyX** [comment]
You are right @praiskup, I tagged only mock-core-configs to the infra tags.

```
[jkadlcik@zeratul copr]$ ./releng/koji-infratag-available --prod mock-core-configs-40.4-1.fc39.noarch.rpm mock-5.6-1.fc39.noarch.rpm
Found: mock-core-configs-40.4-1.fc39.noarch.rpm
Missing: mock-5.6-1.fc39.noarch.rpm
```

Going to tag Mock itself as well.
